### ISSUE TITLE
missing：prompt PROMPT

### ISSUE THREAD

**Issue by juntaosun** [Const-me/Whisper#56]
--prompt PROMPT 
It's not implemented, it doesn't work

**Comment by 12vpn** [comment]
Yes, it's very important for Chinese Support. Without a prompt, we cannot choose the output to be Simplified or Traditional. See this link:[https://github.com/openai/whisper/discussions/277](url)

**Comment by Const-me** [comment]
@juntaosun Implemented in version 1.11 for command-line and power shell.

**Comment by ldcx9** [comment]
> @juntaosun Implemented in version 1.11 for command-line and power shell.

I use version 1.14,but is don't word too;

static int[]? parsePrompt( string str )
{
	if( string.IsNullOrWhiteSpace( str ) )
		return null;
	// TODO: expose whisper_tokenize function, as a method of iModel COM interface
	throw new NotImplementedException();
}
